apps/src/sites/code.org/pages/views/theme_google_analytics.js

```diff
@@ -0,0 +1,11 @@
+import googleAnalyticsReporter from '@cdo/apps/lib/util/GoogleAnalyticsReporter';
+
+const {userAnalyticsDimensions = {}} = window;
```

Comment:
Not familiar with this syntax; what does it do?

Reply:
This is destructuring with a default value. Functionally the same as:

```js
const userAnalyticsDimensions = window.userAnalyticsDimensions ?? {};
```

I didn't realize you could provide default values here, but it looks handy in this case where we can't be certain that the global `window.userAnalyticsDimensions` has been initialized.